src/DiagnosticMessages.ts

```diff
@@ -605,6 +605,21 @@ export let DiagnosticMessages = {
         message: `Missing exception expression after 'throw' keyword`,
         code: 1118,
         severity: DiagnosticSeverity.Error
+    }),
+    missingLeftBracketAfterDimIdentifier: () => ({
+        message: `Missing left bracket after 'dim' identifier`,
```

Comment:
Change function name and message to say "left square bracket" to line up with the way we name our TokenKind (Paren, SquareBracket, CurlyBrace) https://github.com/rokucommunity/brighterscript/blob/master/src/lexer/TokenKind.ts#L5